requirements.yml

```diff
@@ -25,5 +25,5 @@ collections:
     type: git
   - name: https://github.com/ansible-collections/community.crypto
     type: git
-  - name: https://github.com/ansible-collections/kubernetes.core
```

Comment:
I think this is a wrongly solved conflict... ? IIRC we still rely on kubernetes.core collection.